pkg/osvscanner/vulnerability_result.go

```diff
@@ -21,21 +21,18 @@ func buildVulnerabilityResults(
 	packages []scannedPackage,
 	vulnsResp *osv.HydratedBatchedResponse,
 	licensesResp [][]models.License,
-	callAnalysis bool,
-	allPackages bool,
-	licenses bool,
-	licensesAllowlist []string,
+	actions ScannerActions,
 ) models.VulnerabilityResults {
 	output := models.VulnerabilityResults{
 		Results: []models.PackageSource{},
 	}
 	groupedBySource := map[models.SourceInfo][]models.PackageVulns{}
-	if len(licensesAllowlist) == 1 && licensesAllowlist[0] == "" {
-		licensesAllowlist = []string{}
+	if len(actions.ScanLicensesAllowlist) == 1 && actions.ScanLicensesAllowlist[0] == "" {
```

Comment:
I think this check shouldn't be done in the library, but out in the CLI level (e.g. main.go). The library should be able to reasonably expect that all the ScanLicensesAllowlist entries are actual licenses.

Reply:
oops I actually do check it in main.go! but i forgot to remove the check here. I'll remove it.